ddl/db_test.go

```diff
@@ -739,17 +842,32 @@ func (s *testDBSuite) TestAddMultiColumnsIndex(c *C) {
 	s.tk.MustExec("admin check table test")
 }
 
-func (s *testDBSuite) TestAddIndex(c *C) {
-	s.testAddIndex(c, false, "create table test_add_index (c1 bigint, c2 bigint, c3 bigint, primary key(c1))")
+func (s *testDBSuite1) TestAddIndex1(c *C) {
 	s.testAddIndex(c, true, `create table test_add_index (c1 bigint, c2 bigint, c3 bigint, primary key(c1))
 			      partition by range (c1) (
 			      partition p0 values less than (3440),
 			      partition p1 values less than (61440),
 			      partition p2 values less than (122880),
 			      partition p3 values less than (204800),
 			      partition p4 values less than maxvalue)`)
+}
+
+func (s *testDBSuite2) TestAddIndex2(c *C) {
```

Comment:
Is this duplicate with `TestAddIndex1`?

Reply:
No, there are splitting TestAddIndex to different suites

Reply:
every testAddIndex in before `TestAddIndex` is independent.